scripts/push

```diff
@@ -8,7 +8,12 @@ then
 fi
 
 source "$(dirname $0)/version"
-IMAGEID=$( docker images -q openebs/jiva:${VERSION} )
+if [ "$ARCH" == "linux_arm64" ]; then
```

Comment:
Can you make the `linux_amd64` arch default.
```
if arch == amd64, use ${VERSION}
else, use ${XC_ARCH}:${VERSION}
```
This will ease adding more architectures later.

Reply:
Done